src/app/modules/MainScreen/store/actions/index.ts

```diff
@@ -1,27 +1,42 @@
-import { ReturnType } from '@namespace/index';
+import { ReturnType, reduxAsyncAction } from '@namespace/index';
 
 export enum MainScreenTypes {
   INCREMENT = 'INCREMENT',
+  INCREMENT_SUCCESS = 'INCREMENT_SUCCESS',
+  INCREMENT_FAILED = 'INCREMENT_FAILED',
   DECREMENT = 'DECREMENT',
+  DECREMENT_SUCCESS = 'DECREMENT_SUCCESS',
```

Comment:
@eliotmess i think this was just an example, and will not be used later on. In this application there should be the same example as in standard espeo raect app, with loading articles. So when articles example will be incorporated here we will not need it. So maybe focus on recreating in React Native the same integration with artyucles API we ahev in stardard React?

Reply:
Understood.
So firstly I will focus on integration with articles api and afterwards I'll add redux-observable as optional middleware.